src/Elastic.Apm/Api/Database.cs

```diff
@@ -0,0 +1,13 @@
+namespace Elastic.Apm.Api
+{
+	/// <summary>
+	/// An object containing contextual data for database spans.
+	/// It can be attached to an <see cref="ISpan"/> through <see cref="ISpan.Context"/>
+	/// </summary>
+	public class Database
+	{
+		public string Instance { get; set; }
+		public string Statement { get; set; }
+		public string Type { get; set; }
+	}
```

Comment:
Spec at https://github.com/elastic/apm-server/blob/master/docs/spec/spans/span.json has `user` property - was it omitted from this class on purpose?

Reply:
Do you mean "not yet implemented" by APM Server?

Reply:
I mean not yet implemented in the .NET Agent... we just did not add it to the agent. Usually I add field when we also have autoinstrumentation for it.